### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by Eitot**

### CODE REVIEW

File: MMTabBarView/MMTabBarView/MMTabBarView.h [hunk 1]
```diff
@@ -91,7 +91,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Delegate
  */
-@property (weak)   IBOutlet id <MMTabBarViewDelegate> delegate;
+@property (nullable, strong)   IBOutlet id <MMTabBarViewDelegate> delegate;
```
Comment (TAKeanice) [inline MMTabBarView/MMTabBarView/MMTabBarView.h:94]: This will cause retain cycles

**Review by TAKeanice — CHANGES_REQUESTED** [review]
See inline comment (weak delegate)

File: MMTabBarView/MMTabBarView/MMTabBarView.h (hunk 1 above)
Comment (TAKeanice) [reply MMTabBarView/MMTabBarView/MMTabBarView.h:94]: https://github.com/ViennaRSS/MMTabBarView/commit/3d911232d37ab221ecf2ce9a06b27fb960f7934b reinserted strong and removed weak from the ivar. Only the declaration survived your merge ;)

**Comment by barijaona** [comment]
I incorporated changes introduced by this in #37

**Pull request closed without merging**